src/JustEat.StatsD/Buffered/Tags/DataDogTagsFormatter.cs

```diff
@@ -0,0 +1,22 @@
+namespace JustEat.StatsD.Buffered.Tags
+{
+    /// <summary>
+    /// Formats StatsD tags for DataDog, Splunk and CloudWatch.
```

Comment:
As this supports more than one provider, maybe it can have a more generic name?

Reply:
TrailingTagsFormatter for example?

Reply:
That sounds reasonable.